Networking/NetworkingTests/Remote/WordPressSiteRemoteTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+@testable import Networking
+
+final class WordPressSiteRemoteTests: XCTestCase {
+
+    /// Dummy Network Wrapper
+    ///
+    let network = MockNetwork()
+
+    let sampleSiteURL = "https://test.com"
+
+    /// Repeat always!
+    ///
+    override func setUp() {
+        network.removeAllSimulatedResponses()
+    }
+
+    /// Verifies that fetchSiteInfo properly parses the sample response.
+    ///
+    func test_fetchSiteInfo_properly_returns_plugins() async throws {
```

Comment:
Is the name `..._properly_returns_plugins` relevant here?

Reply:
My bad 😅 Fixed in 532ad60f270af9f3f377ba895557695b133154a7.